requirements.txt

```diff
@@ -1,5 +1,6 @@
 git-lfs
-natcap.invest
+#natcap.invest
+git+https://github.com/natcap/invest.git@release/3.12#egg=natcap.invest
```

Comment:
Hey @phargogh, I don't think it makes sense to leave this in here right?

Reply:
Ooo good catch!  Yes, I updated it in https://github.com/natcap/invest.users-guide/commit/30464f4c6f73feebd1b8a02b8e0800b9854ce89d to use `main` instead.  We had previously been using the latest released version of InVEST, but I think `main` might be a better fit.